[2022-11-20T00:45:07.005+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Call_Center_ETL_DAG.insert_customer_data manual__2022-11-20T00:44:38.455644+00:00 [queued]>
[2022-11-20T00:45:07.032+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Call_Center_ETL_DAG.insert_customer_data manual__2022-11-20T00:44:38.455644+00:00 [queued]>
[2022-11-20T00:45:07.033+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-20T00:45:07.035+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-20T00:45:07.036+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-20T00:45:07.070+0000] {taskinstance.py:1383} INFO - Executing <Task(MySqlOperator): insert_customer_data> on 2022-11-20 00:44:38.455644+00:00
[2022-11-20T00:45:07.079+0000] {standard_task_runner.py:55} INFO - Started process 3537 to run task
[2022-11-20T00:45:07.084+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Call_Center_ETL_DAG', 'insert_customer_data', 'manual__2022-11-20T00:44:38.455644+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/call_center_dag.py', '--cfg-path', '/tmp/tmpez7bfj9s']
[2022-11-20T00:45:07.086+0000] {standard_task_runner.py:83} INFO - Job 141: Subtask insert_customer_data
[2022-11-20T00:45:07.173+0000] {task_command.py:376} INFO - Running <TaskInstance: Call_Center_ETL_DAG.insert_customer_data manual__2022-11-20T00:44:38.455644+00:00 [running]> on host 9eba6bc17a61
[2022-11-20T00:45:07.293+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Abdelrhman Yassein
AIRFLOW_CTX_DAG_ID=Call_Center_ETL_DAG
AIRFLOW_CTX_TASK_ID=insert_customer_data
AIRFLOW_CTX_EXECUTION_DATE=2022-11-20T00:44:38.455644+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-20T00:44:38.455644+00:00
[2022-11-20T00:45:07.298+0000] {mysql.py:84} INFO - Executing: 
        INSERT INTO customers(customer_name,city,state, sentiment)
        SELECT 
            customer_name,
            city,
            state,
            sentiment
        FROM
            staging_call
        WHERE
            id IS NOT NULL;
        
[2022-11-20T00:45:07.326+0000] {base.py:71} INFO - Using connection ID 'mysql_conn_id' for task execution.
[2022-11-20T00:45:07.375+0000] {sql.py:315} INFO - Running statement: 
        INSERT INTO customers(customer_name,city,state, sentiment)
        SELECT 
            customer_name,
            city,
            state,
            sentiment
        FROM
            staging_call
        WHERE
            id IS NOT NULL;
        , parameters: None
[2022-11-20T00:45:10.790+0000] {sql.py:324} INFO - Rows affected: 32941
[2022-11-20T00:45:10.909+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=Call_Center_ETL_DAG, task_id=insert_customer_data, execution_date=20221120T004438, start_date=20221120T004507, end_date=20221120T004510
[2022-11-20T00:45:10.965+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-20T00:45:11.008+0000] {taskinstance.py:2623} INFO - 3 downstream tasks scheduled from follow-on schedule check
